src/gateway/understanding-kong/kong-performance-testing.md

```diff
@@ -1,7 +1,236 @@
 ---
-title: Kong Performance testing
-
+title: Kong Gateway Performance Testing
+content_type: explanation
 ---
 
-## PLACEHOLDER Kong Performance Testing
+The {{site.base_gateway}} codebase includes a performance testing framework. It allows Kong developers and users to evaluate the performance of Kong itself as well as 
+bundled or custom plugins, and plot frame graphs to debug performance bottlenecks.
+The framework collects RPS (request per second) and latencies of Kong processing the request
+to represent performance metrics under different workloads.
+
+The framework is implemented as an extension to Kong's integration test suite.
+
+## Installation
+
+The framework uses [busted](https://olivinelabs.com/busted/) and some
+Lua development dependencies of Kong. To set up the environment,
+run `make dev` on your Kong repository to install all Lua dependencies.
+
+## Usage
+
+### Basic usage
+
+Install Lua development dependencies of Kong being installed and invoke each test file with `busted`. The following runs the simple RPS and latency
+test using `docker` driver.
+
+```shell
+PERF_TEST_USE_DAILY_IMAGE=true PERF_TEST_VERSIONS=git:master,git:perf/your-other-branch bin/busted -o gtest spec/04-perf/01-rps/01-simple_spec.lua
+```
+
+User can also run the performance test framework inside a Kong container from scratch, from other development tools
+like [gojira](https://github.com/kong/gojira) or [Pongo](https://github.com/kong/kong-pongo), if they don't have development environment.
+Note to use the `docker` driver inside a container, the Docker daemon socket `/var/run/docker.sock` needs to be mounted inside the container.
```

Comment:
```suggestion
```

Reply:
I learned last week that we don't want to steer customers towards Gojira as a solution.